src/plugin-support.h

```diff
@@ -16,12 +16,15 @@ You should have received a copy of the GNU General Public License along
 with this program. If not, see <https://www.gnu.org/licenses/>
 */
 
-#ifndef PLUGINNAME_H
-#define PLUGINNAME_H
+#pragma once
 
-#define PLUGIN_NAME "@CMAKE_PROJECT_NAME@"
-#define PLUGIN_VERSION "@CMAKE_PROJECT_VERSION@"
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdarg.h>
+#include <string.h>
 
```

Comment:
This would benefit from an
```
#ifdef __cplusplus
extern "C" {
#endif
```
for use in C++-plugins. (it's at least needed to make Windows CI happy, macOS was just fine without it for some reason)